hmpo-lev/email/messages/messages_en.properties

```diff
@@ -0,0 +1,42 @@
+emailVerificationSubject=Welcome to LEV
+emailVerificationLinkTextHtml=Verify your email address
+passwordResetSubject=Reset your LEV password
+passwordResetLinkTextHtml=Reset your LEV password
+emailVerificationBody=Hi {1} {2},\n\n\
```

Comment:
I can't help but feel a lot of this should probably be moved up a level, so that other teams can benefit from the changes... let's see if these translate to any tangible improvements, then look at promotion...

Reply:
The way freemarker seems to work is that new themes override old ones...so sharing bits could be tricky unless they are added as global variables but then need to be referenced in every other theme - as you say though, it's a problem for perhaps further down the line.